app/jobs/get_usps_proofing_results_job.rb

```diff
@@ -0,0 +1,111 @@
+class GetUspsProofingResultsJob < ApplicationJob
+  IPP_STATUS_PASSED = 'In-person passed'
+  IPP_STATUS_FAILED = 'In-person failed'
+  IPP_INCOMPLETE_ERROR_MESSAGE = 'Customer has not been to a post office to complete IPP'
+  IPP_EXPIRED_ERROR_MESSAGE = 'More than 30 days have passed since opt-in to IPP'
+  SUPPORTED_ID_TYPES = [
+    "State driver's license",
+    "State non-driver's identification card",
+  ]
+
+  queue_as :default
+  include GoodJob::ActiveJobExtensions::Concurrency
+
+  good_job_control_concurrency_with(
+    total_limit: 1,
+    key: -> do
+      'get_usps_proofing_results'
+    end,
+  )
+
+  discard_on GoodJob::ActiveJobExtensions::Concurrency::ConcurrencyExceededError
+
+  def perform(_now)
+    return true unless IdentityConfig.store.in_person_proofing_enabled
+
+    proofer = UspsInPersonProofer.new
+
+    InPersonEnrollment.needs_usps_status_check(...5.minutes.ago).each do |enrollment|
+      # Record and commit attempt to check enrollment status to database
+      enrollment.status_check_attempted_at = Time.now
+      enrollment.save
+
+      unique_id = enrollment.usps_unique_id
+
+      response = nil
+      begin
+        response = proofer.request_proofing_results unique_id, enrollment.enrollment_code
```

Comment:
I notice this PR omits a lot of parentheses? I know that Seattle ruby style is to use as few as possible, but in this case I think this would be much clearer with them:
```suggestion
        response = proofer.request_proofing_results(unique_id, enrollment.enrollment_code)
```

Reply:
addressed in 9822bcb65583055dab9e463a944f83fc33c8378d and 47454c1d6a4451f2d1b1202212ba7e37d036c98a